decidim-core/app/views/kaminari/decidim/_paginator.html.erb

```diff
@@ -8,19 +8,19 @@ paginator:     the paginator that renders the pagination tags inside -%>
 
 <% if total_pages > 1 %>
   <nav aria-label="<%= t("views.pagination.pagination") %>">
-    <ul class="pagination text-center">
-      <%= paginator.first_page_tag unless current_page.first? %>
```

Comment:
I find it very sad if we lose the first and last links in the pagination because they really improve usability. Many times you want to jump into the first or last page of the pagiation.

These are also included in the W3C design system (which they use for their sites):
https://design-system.w3.org/components/pagination.html

Alternative implementations include showing the first/last pages always as numbers in the pagination list, such as the bounded example here:
https://designsystem.digital.gov/components/pagination/